modules/nw-ingress-controller-endpoint-publishing-strategies.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-controller-endpoint-publishing-strategies_{context}"]
+= Ingress controller endpoint publishing strategy
+
+An Ingress controller with the `HostNetwork` endpoint publishing strategy can have only one pod replica per node. If you want _n_ replicas, you need at least _n_ nodes where those replicas can be scheduled because each pod replica requests ports `80` and `443` on the node host where it is scheduled. So a replica cannot be scheduled to a node if another pod on the same node is using those ports.
```

Comment:
s/pod/Pod
s/need/must use

I'd change the split of the sentence:
s/scheduled because/scheduled. Because
s/scheduled. So a replica/scheduled, a replica

s/is using/uses